client/src_editor/Editor_Blocks.re

```diff
@@ -426,13 +542,13 @@ let make =
     <>
       state.blocks
       ->(
-          Belt.Array.mapU((. {b_id, b_data}) =>
+          Belt.Array.mapU((. {b_id, b_data, b_deleted}) =>
             switch (b_data) {
             | B_Code({bc_value, bc_widgets, bc_firstLineNumber}) =>
               <div key=b_id id=b_id className="block__container">
                 <div className="source-editor">
                   <Editor_CodeBlock
-                    value=bc_value
+                    value=(b_deleted ? warning(lang, BTyp_Code) : bc_value)
```

Comment:
👇  I ideally would like the temporary blocks to be readonly, however the following does not work:
```reason
readOnly=(b_deleted ? true : readOnly)
```
which is odd, because the same ternary conditions work correctly here. Any ideas as to why this doesn't work? I'm probably missing something pretty obvious, but it seems like this should be fine.

Reply:
I'm not sure what is going on either. readOnly should work otherwise it's a bug somewhere. Quick thought: if you're going to make it readOnly, why don't we just drop the editor completely and render something else?

Reply:
That's a thought... we could just render a div with the same content and the controls underneath to restore. This would reduce a small amount of complexity with rendering different style comments for RE vs. ML and markdown vs. code for different kinds of blocks.

I do like the current style of the UI with the temp block being part of the code, though.

Reply:
As a sidenote, readOnly works fine when I pass just `true`

Reply:
Oh. So what do you mean by does not work? It doesn't compile right ?

Reply:
It doesn't actually make the block readonly. Sorry I should've clarified that.